src/main/java/ro/societateahermes/backendservice/controller/controllerImplementation/CDMemberControllerImplementation.java

```diff
@@ -0,0 +1,65 @@
+package ro.societateahermes.backendservice.controller.controllerImplementation;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+
+import ro.societateahermes.backendservice.controller.CDMemberControllerInterface;
+import ro.societateahermes.backendservice.entities.CDMember;
+import ro.societateahermes.backendservice.entities.DTO.CDMemberDTO;
+import ro.societateahermes.backendservice.service.serviceImplementation.CDMemberServiceImplementation;
+
+import javax.validation.Valid;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/cdMember")
+public class CDMemberControllerImplementation implements CDMemberControllerInterface {
+    private final CDMemberServiceImplementation CDMemberService;
+
+    public CDMemberControllerImplementation(CDMemberServiceImplementation serviceImplementation) {
+        CDMemberService = serviceImplementation;
+    }
+
+    @PostMapping("/CD")
```

Comment:
url-ul ar trebui sa fie ceva de genul /cd/save

Reply:
@McShenanigans  Actually, not. Url-ul de baza fiind cdMember (recomand sa fie cd-member, ca e mai lizibil), facand un post pe cel de baza, se intelege ca se face un save pentru un CD member.  Deci ar trebui sa fie chiar si fara /CD, doar POST /cdMember

@mircea216  in cadrul url-urilor nu ar trebui folosite litere mari, cand ai mai multe cuvinte, incearca sa le desparti cu -

Reply:
@McShenanigans @fujinuji in the end, am lasat url urile fara save/update and so on, iar cdMember l am inlocuit cu cd-member and everything seems a bit more clear